### DISCUSSION TITLE
Create a new tab/window

### DISCUSSION THREAD

**Discussion by JuGid** [php-webdriver/php-webdriver#862] [Q&A] [answered]
Hi,

I think the doc isn't up-to-date because it specifies this to create new window/tab

```
// Try to open new window
$driver->switchTo()->newWindow(WebDriverTargetLocator::WINDOW_TYPE_WINDOW);
// Try to open new tab
$driver->switchTo()->newWindow(WebDriverTargetLocator::WINDOW_TYPE_TAB);
```

Since there is no constant variable in WebDrivertargetLocator class, how can I achieve this (especially for tabs) ?

**Comment by OndraM** [comment, marked as answer]
Hi, the doc is up-to-date with current main branch which includes this new feature, however, it was not yet released.

You can **temporarily** use dev-main as your composer dependency, or wait a few days until we tag new release (version 1.10.0).

Until then, you can only use

```$driver->newWindow();```

**Reply by JuGid** [reply]
Thanks for your fast and detailed answer. I'm waiting for this new release !
